### DISCUSSION TITLE
Inputs style inside popovers

### DISCUSSION THREAD

**Discussion by OmranAbazid** [carbon-design-system/carbon#14672] [Help] [answered]
Inputs inside popovers currently inherit the background color of the popover instead of contrasting with it.
This is not the case for other container elements such as modals.

![Screenshot 2023-09-18 at 2 03 34 PM](https://github.com/carbon-design-system/carbon/assets/9063094/7ad0d477-9545-4851-bc7b-103b9425b10e)

Here is an example:
https://codesandbox.io/s/funny-kowalevski-f8m4c6?file=/index.js

Is this behavior intended? Or is it forbidden to use inputs inside popovers?

**Comment by tay1orjones** [comment, marked as answer]
Hey @OmranAbazid! Popover doesn't redefine the layer by default. If you wrap the popover contents in a `<Layer>` you should get the proper result in regards to the background styling.
